coding-concepts.js

```diff
@@ -6,54 +6,59 @@
 // Then, uncomment the console.log() and verify the output. Briefly explain why your initial answer was correct or incorrect.
 
 // --------------------INSTRUCTOR EXAMPLE: What will this log?
-const colors = ["tangerine", "magenta", "lilac", "daffodil"]
-// console.log(colors.push("indigo"))
+// const colors = ["tangerine", "magenta", "lilac", "daffodil"]
+// // console.log(colors.push("indigo"))
 
-// a) Your answer:
-// b) Verify and explain:
+// // a) Your answer:
+// // b) Verify and explain:
 
 
 // --------------------1) What will this log?
 
-const cohort = "LEARN 2022"
+// const cohort = "LEARN 2022"
 // console.log(cohort.length)
 
-// a) Your answer:
-// b) Verify and explain:
+
+// a) Your answer: The answer to Learn 2022 should be 10 becuase the string is five character, the space which is data and 2022
+// b) Verify and explain: After running the code the answer was 10
 
 
 // --------------------2) What will this log?
 
-const greeting = "Hello World!"
+// const greeting = "Hello World!"
 // console.log(greeting[4])
 
-// a) Your answer:
-// b) Verify and explain:
+
+// a) Your answer: The answer should be "o" in Hello because the index is 4
+// b) Verify and explain: The index gives the position of the first occurrency and that first occurence would be "o"
 
 
 // --------------------3) What will this log?
 
-const languages = ["JavaScript", "Ruby", "Python", "C++"]
-const index = 1
+// const languages = ["JavaScript", "Ruby", "Python", "C++"]
+// const index = 1
 // console.log(languages[index])
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: Since index is 1 the answer should be Ruby
+// b) Verify and explain: The correct answer is Ruby based on the same index explanation in question 2
 
 
 // --------------------4) What will this log?
 
-const weekendDays = ["saturday", "sunday"]
+// const weekendDays = ("saturday", "sunday")
 // console.log(weekendDays.toUpperCase())
 
-// a) Your answer:
-// b) Verify and explain:
+// a) Your answer: I expected to get Saturday in lower case and Sunday in caps.
+// b) Verify and explain: I removed the square bracketts and repalced it with parenthesis but only Sunday in caps returned. Probably had to type out the command instead of hitting the up arrow.
```

Comment:
The real goal here was to understand that some methods only work on arrays (or lists of data) and some methods only work on strings, etc.